src/main/java/org/openlmis/fulfillment/web/OrderController.java

```diff
@@ -306,6 +307,27 @@ public NumberOfOrdersData getOrdersData() {
     return ordersData;
   }
 
+  /**
+   * Get information about number of orders of each status for user's home facility.
+   *
+   * @return Orders statistics data.
+   */
+  @GetMapping("/orders/statusesStatsData")
+  @ResponseBody
+  public OrderStatsData getOrderStatusesStatsData() {
+    Profiler profiler = new Profiler("COUNT_ORDERS_STATISTICS_DATA");
+    profiler.setLogger(XLOGGER);
+    UUID facilityId =  authenticationHelper.getCurrentUser().getHomeFacilityId();
+    if (facilityId == null) {
+      return new OrderStatsData();
+    }
+    profiler.start("COUNT_ORDERS_STATUSES_DATA_IN_SERVICE");
+    OrderStatsData orderStatusesData = orderService.getStatusesStatsData(facilityId);
```

Comment:
What do you think about changing profiler name to something like "GET_ORDER_STATISTICS_DATA" and the second step name to just "COUNT_ORDERS_STATUSES_DATA"?

Reply:
I changed the names as suggested.